spec/src/main/doc/core/spi.asciidoc

```diff
@@ -1076,19 +1061,19 @@ public interface ProcessInjectionTarget<X> {
 }
 ----
 
-* +getAnnotatedType()+ returns the +AnnotatedType+ representing the managed bean class, session bean class or other Java EE component class supporting injection.
+* +getAnnotatedType()+ returns the +AnnotatedType+ representing the bean class or other component class supporting injection.
 * +getInjectionTarget()+ returns the +InjectionTarget+ object that will be used by the container to perform injection.
 * +setInjectionTarget()+ replaces the +InjectionTarget+.
 * +addDefinitionError()+ registers a definition error with the container, causing the container to abort deployment after bean discovery is complete.
 
 
-Any observer of this event is permitted to wrap and/or replace the +InjectionTarget+. The container must use the final value of this property, after all observers have been called, whenever it performs injection upon the managed bean, session bean or other Java EE component class supporting injection.
+Any observer of this event is permitted to wrap and/or replace the +InjectionTarget+. The container must use the final value of this property, after all observers have been called, whenever it performs injection upon the bean or other component class supporting injection.
 
-For example, this observer decorates the +InjectionTarget+ for all servlets.
+For example, this observer decorates the +InjectionTarget+ for all Maps.
```

Comment:
A Map is not a good substitute here. Map is a generic type and raw references to generic types are deprecated in Java since version 5.